staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/types.go

```diff
@@ -304,6 +309,11 @@ const (
 	NonStructuralSchema CustomResourceDefinitionConditionType = "NonStructuralSchema"
 	// Terminating means that the CustomResourceDefinition has been deleted and is cleaning up.
 	Terminating CustomResourceDefinitionConditionType = "Terminating"
+	// KubeAPIApproved indicates that an API in *.k8s.io or *.kubernetes.io is or is not approved.  For CRDs
```

Comment:
I think you can't use the name "...Approved" to also mean "...or is not approved", that will confuse people. Maybe "ApprovalStatusKnown"? "ApprovalPolicyConformant"?

Also no abbreviations, s/Kube/Kubernetes/